pkg/info/infopb/rpc.proto

```diff
@@ -58,10 +58,14 @@ message StoreInfo {
 message RulesInfo {
 }
 
-/// MetricMetadataInfo holds the metadata related to Metadata API exposed by the component.
+/// MetricMetadataInfo holds the metadata related to Metric Metadata API exposed by the component.
 message MetricMetadataInfo {
 }
 
+/// TargetMetadataInfo holds the metadata related to Target Metadata API exposed by the component.
+message TargetMetadataInfo {
```

Comment:
I am wondering is it better to combine this and `MetricMetadataInfo` to one `MetadataInfo` field, in the future?
As they are both belonging to `metadatapb`, but different services.

Reply:
good point, resolved